plugin/client.py

```diff
@@ -218,110 +215,6 @@ def patch_markdown_content(self, content: str) -> str:
         content = re.sub(r"\n:deprecated:", r"\n⚠️ __Deprecated:__", content)
         return content
 
-    def detect_st_py_ver(self, dev_environment: str) -> tuple[int, int]:
-        default = (3, 3)
-
-        if dev_environment == "sublime_text_33":
-            return (3, 3)
-        if dev_environment == "sublime_text_38":
-            return (3, 8)
-        if dev_environment == "sublime_text":
-            if not ((session := self.weaksession()) and (workspace_folders := session.get_workspace_folders())):
-                return default
-            # ST auto uses py38 for files in "Packages/User/"
-            if (first_folder := Path(workspace_folders[0].path).resolve()) == Path(sublime.packages_path()) / "User":
-                return (3, 8)
-            # the project wants to use py38
-            try:
-                if (first_folder / ".python-version").read_bytes().strip() == b"3.8":
-                    return (3, 8)
-            except Exception:
-                pass
-            return default
-
-        raise ValueError(f'Invalid "dev_environment" setting: {dev_environment}')
-
-    def find_package_dependency_dirs(self, py_ver: tuple[int, int] = (3, 3)) -> list[str]:
-        dep_dirs = sys.path.copy()
-
-        # replace paths for target Python version
-        # @see https://github.com/sublimelsp/LSP-pyright/issues/28
-        re_pattern = re.compile(r"(python3\.?)[38]", flags=re.IGNORECASE)
-        re_replacement = r"\g<1>8" if py_ver == (3, 8) else r"\g<1>3"
-        dep_dirs = [re_pattern.sub(re_replacement, dep_dir) for dep_dir in dep_dirs]
-
-        # move the "Packages/" to the last
-        # @see https://github.com/sublimelsp/LSP-pyright/pull/26#discussion_r520747708
-        packages_path = sublime.packages_path()
-        dep_dirs.remove(packages_path)
-        dep_dirs.append(packages_path)
-
-        # sublime stubs - add as first
-        if py_ver == (3, 3) and (server_dir := self._server_directory_path()):
-            dep_dirs.insert(0, os.path.join(server_dir, "resources", "typings", "sublime_text_py33"))
-
-        return list(filter(os.path.isdir, dep_dirs))
-
-    @classmethod
-    def _print_print_sys_paths(cls, sink: Callable[[str], None]) -> None:
-        sink("import sys")
-        sink("import json")
-        sink('json.dump({"executable": sys.executable, "paths": sys.path}, sys.stdout)')
-
-    @classmethod
-    def _get_dev_environment_binary(cls, settings: DottedDict, name: str) -> str:
-        return settings.get(f"settings.dev_environment.{name}.binary") or name
```

Comment:
During refactoring, I found this should be `pyright.dev_environment.{name}.binary"`. So it shouldn't be working at this moment...